components/com_content/src/View/Category/FeedView.php

```diff
@@ -53,9 +53,12 @@ protected function reconcileNames($item)
         $item->description = '';
         $obj               = json_decode($item->images);
 
+        // Set feed image to image_intro or if that's empty, to image_fulltext
         if (!empty($obj->image_intro)) {
             $item->description = '<p>' . HTMLHelper::_('image', $obj->image_intro, $obj->image_intro_alt) . '</p>';
-        }
+		} elseif (!empty($obj->image_fulltext)) {
+			$item->description = '<p>' . HTMLHelper::_('image', $obj->image_fulltext, $obj->image_fulltext_alt) . '</p>';
+		}
```

Comment:
```suggestion
        } elseif (!empty($obj->image_fulltext)) {
            $item->description = '<p>' . HTMLHelper::_('image', $obj->image_fulltext, $obj->image_fulltext_alt) . '</p>';
        }
```
change the tabs to spaces